### PR TITLE
Updated AddItemCommand such that it checks for existing person in PersonAmountBook

### PR DESCRIPTION
**Pull request by ailanthustng**

### CODE REVIEW

File: src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java [hunk 1]
```diff
@@ -30,22 +31,37 @@
...
         model.addEntry(toBeAdded);
+        for (Person person : persons) {
+            if (model.hasPerson(person)) {
+                model.addAmount(person, amount);
+                // person.addAmount(amount); //not sure if necessary
+            } else {
+                // add person, then add amount to that person
+                model.addPerson(person);
+                model.addAmount(person, amount);
+                // person.addAmount(amount); //not sure if necessary
```
Comment (sheexiong) [inline src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java:62]: I think this is kind of redundant seems the model already add the person with the amount, the person should not add amount again.

**Review by sheexiong — APPROVED** [review]
I think the overall code is alright, I will continue working on the listamount based on the given code.

File: src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java (hunk 1 above)
Comment (ailanthustng) [reply src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java:62]: `addPerson` doesn't add amount. It simply creates the person in the personAmountBook

Comment (sheexiong) [reply src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java:62]: Then how about `addAmount` ? Did this method add the person with the amount?

Comment (ailanthustng) [reply src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java:62]: `addAmount` simply adds the amount to the person

**Pull request merged**